yt_dlp/extractor/lastfm.py

```diff
@@ -44,9 +44,21 @@ class LastFMPlaylistIE(LastFMPlaylistBaseIE):
     }, {
         'url': 'https://www.last.fm/music/Oasis',
         'only_matching': True,
+    }, {
+        'url': 'https://www.last.fm/music/Oasis/',
+        'only_matching': True,
+    }, {
+        'url': 'https://www.last.fm/music/Oasis?top_tracks_date_preset=ALL#top-tracks',
+        'only_matching': True,
```

Comment:
This is common issue for any extractor that tries to put `$` at the end, so tests were not really needed, but 🤷